reference/forms/types/enum.rst

```diff
@@ -52,6 +52,40 @@ This will display a ``<select>`` tag with the three possible values defined in
 the ``TextAlign`` enum. Use the `expanded`_ and `multiple`_ options to display
 these values as ``<input type="checkbox">`` or ``<input type="radio">``.
 
+Since the label displayed in the ``<select>`` options is the enum name, you might sometimes
+want more flexibility as PHP strongly restricts the usable characters for those.
+You could do this by implementing a function in your enum class which returns a label 
+or even a translation string for each possible enum::
+
+    // src/Config/TextAlign.php
+    namespace App\Config;
+
+    enum TextAlign: string
+    {
+        case Left = 'Left/Start aligned';
+        case Center = 'Center/Middle aligned';
+        case Right = 'Right/End aligned';
+        
+        public function label(): string
```

Comment:
Adding this method to the enum class starts mixing data and presentation which I would not advise. Let's configure the choice label using a closure instead (see comment below).